.prettierrc

```diff
@@ -0,0 +1,14 @@
+{
+  "tabWidth": 4,
```

Comment:
Is it ironic that this file is using a tab width of 2? :D

Reply:
like rain on your wedding day: 
<img width="610" alt="image" src="https://github.com/technologiestiftung/innovationserhebung/assets/50147356/acf8b390-6845-436c-af33-c35f72bb09d9">